pkg/logs/storefailed.go

```diff
@@ -0,0 +1,9 @@
+package logs
+
+// StoreFailed indicates we were unable to persist scan results for an
+// async job.
+type StoreFailed struct {
+	Message    string `logevent:"message,default=store-failed"`
+	Reason     string `logevent:"reason"`
+	TargetHost string `logevent:"target_host"`
```

Comment:
Nit: This is the nit-est of nits, but all the other log events are ordered `Message` -> `TargetHost` -> `Reason`